rel/files/ejabberd.cfg

```diff
@@ -662,18 +662,15 @@
                            % default: 600 seconds
                            % {resume_timeout, 600}
                           ]},
-%  {mod_muc_light, [
-%                   {host, "muclight.@HOST@"}
-%                  ]},
-  {mod_muc, [
```

Comment:
Do we really want to disable `mod_muc` by default? I know one can enable it in config file, but most users who just want to try it will probably never touch config file.